app/apps/planner/models.py

```diff
@@ -273,6 +278,7 @@ def get_cases_query_params(self):
                 "schedule_week_segment": self.week_segments,
                 "project": self.project_ids,
                 "reason": self.reasons,
+                "district": self.districts if self.districts is not None else [],
```

Comment:
Kan dit mogelijk opgelost worden met een default op het model ? Het staat ook nog ergens anders in de code.
Wel opletten met de default: 
If you give the field a [default](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.Field.default), ensure it’s a callable such as list (for an empty default) or a callable that returns a list (such as a function). Incorrectly using default=[] creates a mutable default that is shared between all instances of ArrayField.

Reply:
Je hebt gelijk, defaults zouden hier goed zijn, heb daar in het verleden wel last mee gehad, ook callables. Ik ga nog even kijken hoe anders, waarschijnlijk is None hier ook goed, daar requests ook goe mee overweg.